apps/docker/Dockerfile

```diff
@@ -1,38 +1,37 @@
-# Librelingo is not compatible with Node 16, using Node 14.18.3 image instead
-FROM node:14.18.3 
+FROM archlinux:base-20220227.0.49015
 
-# Setup Librelingo
+# Setup packages
+RUN pacman -Syu --noconfirm git nodejs-lts-fermium gcc yarn python3 python-pip
+
+# Setup Poetry and LibreLingo Python Packages
+RUN curl -sSL https://install.python-poetry.org | python3 - && \
+  cp /root/.local/bin/poetry /bin && \
+  pip install librelingo-yaml-loader && \
+  pip install librelingo-json-export
+  
+# Git clone LibreLingo repo
 RUN git clone https://github.com/LibreLingo/LibreLingo.git
-WORKDIR /LibreLingo
-RUN yarn set version classic && \
+  
+# Setup LibreLingo 
+RUN cd /LibreLingo && \
+  yarn set version classic && \
   yarn
```

Comment:
this should probably be

```bash
yarn install --frozen-lockfile
```

Reply:
I will change that when I have the chance. I have not much experience with yarn, so I didn't know. Thanks!

Reply:
Changed!

Reply:
amazing!